docs/csharp/language-reference/builtin-types/ref-struct.md

```diff
@@ -30,15 +28,32 @@ To declare a `ref struct` as `readonly`, combine the `readonly` and `ref` modifi
 
 In .NET, examples of a `ref struct` are <xref:System.Span%601?displayProperty=nameWithType> and <xref:System.ReadOnlySpan%601?displayProperty=nameWithType>.
 
+## `ref` fields
+
+Beginning with C# 11, you can declare a `ref` field in a `ref struct`, as the following example shows:
+
+:::code language="csharp" source="snippets/shared/StructType.cs" id="SnippetRefField":::
+
+A `ref` field may have the `null` value. Use the <xref:System.Runtime.CompilerServices.Unsafe.IsNullRef%60%601?displayProperty=nameWithType> method to determine if a `ref` field is `null`.
```

Comment:
@BillWagner could you please help me to fix this xref to https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.unsafe.isnullref?view=net-6.0